package.json

```diff
@@ -95,6 +95,7 @@
     "istanbul-reports": "1.1.4",
     "jest": "^23.6.0",
     "jsdom": "^11.12.0",
+    "json-rules-engine-simplified": "^0.1.17",
```

Comment:
This will probably increase the bundle size of the lib. Can we check this and maybe just add to dev dependency?

Reply:
Yep those are in devDependencies